### DISCUSSION TITLE
WiFi connection failed on ESP32-S3FH4R2

### DISCUSSION THREAD

**Discussion by 28106830** [nanoframework/Home#1553] [Q&A]
### Target name(s)

ESP32-S3FH4R2  PICO

### Firmware version

ESP32_S3-1.12.1.3.zip

### Description

使用ESP32-S3FH4R2上时，使用wifi示例，到

**Comment by josesimoes** [comment]
Moving this to discussions as this seems to be a question, not reporting a confirmed issue.
Please use English for communication.

**Comment by 28106830** [comment]
This issue has been resolved, it is a batch issue with the motherboard
